webplugin/js/app/mck-ringtone-service.js

```diff
@@ -3,7 +3,8 @@
             _this.loadRingTone = function (url,notificationtoneoption) {
                 return new Howl({
                     src: [url],
-                    loop: notificationtoneoption.loop
+                    loop: notificationtoneoption.loop,
+                    html5: true
```

Comment:
Fixed cross-origin error coming because of Howler plugin. For more information refer to below link :
goldfire/howler.js#530